STPopupPreview/STPopupPreviewRecognizer.m

```diff
@@ -318,7 +318,10 @@ - (void)gestureAction:(UIGestureRecognizer *)gesture
             _popupController.hidesCloseButton = YES;
             if (NSClassFromString(@"UIVisualEffectView")) {
                 UIBlurEffect *blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleLight];
-                _popupController.backgroundView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
+                UIView *effectView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
```

Comment:
I checked out the demo, the alpha animation of background view doesn't seem right.

Reply:
I see..., it is broken on 10, but not on 11, am I right?